content/exchange/artifacts/Trawler.yaml

```diff
@@ -0,0 +1,58 @@
+name: Windows.Forensics.Trawler
+description: |
+  Trawler [https://github.com/joeavanzato/Trawler] is a PowerShell script designed to help Incident
+  Responders rapidly identify potential adversary persistence mechanisms on Windows.  It is similar
+  in nature to PersistenceSniper with additional targeted checks as well as the capability to operate
+  against a 'dead box' (mounted drive). The output is simplified compared to PersistenceSniper, providing
+  the user with the details needed to kick off an investigation into any identified mechanisms.
+  Think of these tools as autoruns on steroids.
+
+  Please ensure the Velociraptor binary (and child powershell process) are excluded in any EDR/AV products.
+
+author: Joe Avanzato
+
+parameters:
+  - name: UploadResults
+    type: bool
+    default: true
+
+tools:
+  - name: Trawler
+    url: https://github.com/joeavanzato/Trawler/releases/download/1_0_0/trawler_release_v1_0_0.zip
```

Comment:
can we pin the hash?

Reply:
Done, let me know if this was not what you meant.  Thanks.